types/preview/@ember/array/-private/native-array.d.ts

```diff
@@ -4,10 +4,11 @@ declare module '@ember/array/-private/native-array' {
   import MutableArray from '@ember/array/mutable';
   import Mixin from '@ember/object/mixin';
 
-  // Get an alias to the global Array type to use in inner scope below.
+  // Get aliases to the global Array types to use in inner scope below.
```

Comment:
And we can revert this.

```suggestion
  // Get an alias to the global Array type to use in inner scope below.
```